addons/mail/static/src/core/common/chat_bubble.xml

```diff
@@ -16,11 +16,16 @@
     <t t-name="mail.ChatBubblePreview">
         <div class="o-mail-ChatBubble-preview o-mail-ChatBubble-menu px-0 py-1 shadow-sm border border-secondary bg-100 rounded">
             <div class="text-truncate base-fs fw-bolder mb-0 mx-2" t-esc="props.chatWindow.displayName"/>
-            <div t-if="previewContent" class="text-truncate small mx-2 text-muted">
+            <t t-set="message" t-value="thread?.newestPersistentOfAllMessage" />
+            <div t-if="previewContent or message?.hasOnlyAttachments" class="text-truncate small mx-2 text-muted">
                 <t t-call="mail.message_preview_prefix">
-                    <t t-set="message" t-value="thread?.newestPersistentOfAllMessage"/>
+                    <t t-set="message" t-value="message"/>
+                </t>
+                <t t-if="previewContent" t-out="previewContent" />
+                <t t-else="">
+                    <i class="fa me-1" t-att-class="message.messagePreviewIcon"/>
+                    <t t-out="message.messagePreviewText" />
```

Comment:
`previewContent` should take into account `hasOnlyAttachments`, so the `t-if` on whole `<div>` keeps same single expression.

Inner `t-if="previewContent" t-else=""` should be replaced by `t-if="hasOnlyAttachments" t-else=""` instead.

Reasoning: `previewContent` wording should cover all preview content, including atachments only. This doesn't make semantically sense this is only for text.

Reply:
`ChatBubble.previewContent` becomes an alias to `mostRecentMessage.previewText`.
Could rename `previewContent` to `previewText` for consistent naming.